test/wallet/wallet-unit.js

```diff
@@ -32,51 +43,183 @@ describe('Wallet Unit Tests', () => {
 
       {
         // Should instatiate from String.
-        const wallet = await wdb.create({ master: xprv });
+        const walletdb = await wdb.create({ master: xprv });
```

Comment:
`wdb.create` creates an instance of a `Wallet` not a `WalletDB`.

Reply:
Updated this, good catch